### PR TITLE
Add tag for sending tensors to crypten

### PR DESCRIPTION
**Pull request by gmuraru**
Added ```self``` such that we can reference the worker in the ```toy_func``` and use the tag to work with the specific data.

A possible fix for #2986

### CODE REVIEW

File: syft/frameworks/crypten/context.py [hunk 1]
```diff
@@ -79,12 +82,30 @@ def _send_party_info(worker, rank, msg, return_values):
...
 
 
-def toy_func():
-    # Toy function to be called by each party
-    alice_t = crypten.cryptensor([73, 81], src=0)
-    bob_t = crypten.cryptensor([90, 100], src=1)
-    out = bob_t.get_plain_text()
-    return out.tolist()  # issues with putting torch tensors into queues
+def toy_func(*args, **kwargs):
+    # !! This might work only because we have virtual workers
```
Comment (youben11) [inline syft/frameworks/crypten/context.py]: We shouldn't perform workers operations on crypten functions I think. We should try at best to make those operations transparent by handling them via function decorators or our own crypten.load function

Comment (gmuraru) [reply syft/frameworks/crypten/context.py]: I will look at the ```crypten.load``` how it works and try something from there

File: syft/frameworks/crypten/context.py [hunk 2]
```diff
@@ -79,12 +83,12 @@ def _send_party_info(worker, rank, msg, return_values):
     return_values[rank] = response.contents
 
 
-def toy_func():
-    # Toy function to be called by each party
-    alice_t = crypten.cryptensor([73, 81], src=0)
-    bob_t = crypten.cryptensor([90, 100], src=1)
-    out = bob_t.get_plain_text()
-    return out.tolist()  # issues with putting torch tensors into queues
+def toy_func(*args, **kwargs):
```
[the remark this answers is not in the document]
Comment (gmuraru) [reply syft/frameworks/crypten/context.py]: Done :)

File: syft/frameworks/crypten/__init__.py [hunk 3]
```diff
@@ -1,4 +1,46 @@
...
+
+        # file contains torch.tensor
+        if torch.is_tensor(result):
+            # Broadcast load type
+            load_type = torch.tensor(0, dtype=torch.long)
+            comm.get().broadcast(load_type, src=src)
+
+            # Broadcast size to other parties.
+            dim = torch.tensor(result.dim(), dtype=torch.long)
+            size = torch.tensor(result.size(), dtype=torch.long)
```
Comment (LaRiffle) [inline syft/frameworks/crypten/__init__.py]: I'm curious why you had to provide the dim + the size: is this because the receving part:
```
# Receive size from source party
            dim = torch.empty(size=(), dtype=torch.long)
            comm.get().broadcast(dim, src=src)
```
should have a fixed size in advance?

Comment (youben11) [reply syft/frameworks/crypten/__init__.py]: That's because only the src party knows the dimensions and size of the tensor to be encrypted, and the shares should be created accordingly, this this information should be sent to all the other parties via brodcast.
